assets/scss/components/_sortable-table.scss

```diff
@@ -1,7 +1,7 @@
 [aria-sort] {
   a,
   button {
-    display: inline-block;
+    display: inline-flex;
```

Comment:
Some of the existing styles for the client-side sort use `inline-flex` on `:hover` for the buttons. Using it here prevents a tiny jump of the button when hovering.